products/sle/main.pm

```diff
@@ -458,6 +458,9 @@ sub load_inst_tests() {
             loadtest 'installation/partitioning_full_lvm';
         }
         if (get_var("FILESYSTEM")) {
+            if (get_var('PARTITIONING_WARNINGS')) {
+                loadtest 'installation/partitioning_warnings';
+            }
```

Comment:
well, the same as for the yast development team: Their changes should be in Factory and therefore Tumbleweed. Therefore we should make sure it is tested on Tumbleweed. Most of the features in fate will be selected for particular SLE versions but the changes still go into Tumbleweed.